.github/workflows/build.yml

```
@@ -19,4 +19,86 @@ jobs:
     - name: Build the project
-      run: dotnet build TestStream.Runner/TestStream.Runner.csproj
\ No newline at end of file
+      run: dotnet build TestStream.Runner/TestStream.Runner.csproj
+    
+    - name: Trigger Azure DevOps Pipeline
+      env:
+        AZURE_DEVOPS_ORG: nanoframework
+        AZURE_DEVOPS_PROJECT: nanoFramework.IoT.TestStream
+        AZURE_DEVOPS_PIPELINE_ID: 111
+        AZURE_POOL_NAME: TestStream
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:19:67: Double quote to prevent globbing and word splitting

(shellcheck)